Exiled.API/Features/Map.cs

```diff
@@ -31,6 +31,18 @@ public static class Map
         private static readonly List<Camera079> CamerasValue = new List<Camera079>(250);
         private static readonly List<Lift> LiftsValue = new List<Lift>(10);
         private static readonly List<TeslaGate> TeslasValue = new List<TeslaGate>(10);
+        private static readonly Lazy<Ragdoll.Info> LazyDefaultRagdollOwner = new Lazy<Ragdoll.Info>(() =>
```

Comment:
You don't need the lazy instancing, assign the value to the property directly

Reply:
You're right, but I was thinking about Galaxy's own words of "EXILED doesn't bring any bloat unless a plugin requires it" and stuff. I know it's just a few bytes, but this implementation followed that mindset

Reply:
Yep, he says so, but it's because we don't create a new Player instance every time we call an event.